README.md

```
@@ -11,26 +11,75 @@
+
 **Codesirius**
-Codesirius is a platform to arrange interview sessions with live coding. It is aimed to facilitate the interview process for both the interviewer and the interviewee.
-It has also problem archive where users can practice problems and can see the solutions of the problems. Users can also submit their solutions and can see the results.
+Codesirius is a platform to arrange interview sessions with live coding. It is aimed to
+facilitate the interview process for both the interviewer and the interviewee.
+It has also problem archive where users can practice problems and can see the solutions
+of the problems. Users can also submit their solutions and can see the results.
+
+## Tech Stack
+
+### Frontend  
```

markdownlint-cli2 (0.17.2)

Headings should be surrounded by blank lines
Expected: 1; Actual: 0; Below

(MD022, blanks-around-headings)